packages/@azure/storage/blob/package.json

```diff
@@ -63,7 +63,7 @@
     "typescript": "^3.2.2"
   },
   "scripts": {
-    "test": "npm run test:node && npm run test:browser",
+    "unit": "npm run build && npm run test:node && npm run test:browser",
```

Comment:
I believe additional work is required to publish results of the browser tests to DevOps.  Currently, I believe a browser test failure will cause the build to fail, however the build does not have per-test level results for the browser tests, like it does for the node tests.

Reply:
#1749